packages/select/README.md

```diff
@@ -256,7 +256,7 @@ Name                                          | Description
 | `opened`   | `mwc-select-surface` | none              | Fired menu opens.    |
 | `closed`   | `mwc-select-surface` | none              | Fired menu closes.   |
 | `action`   | `mwc-list`           | `ActionDetail`*   | Fired when a selection has been made via click or keyboard action. |
-| `selected` | `mwc-list`           | `SelectedDetail`* | Fired when a selection has been made. `index` is the selected index (will  be of type `Set<number>` if multi and `number` if single), and `diff` (of type `IndexDiff`**) represents the diff of added and removed indices from previous selection. |
+| `selected` | `mwc-list`           | `SelectedDetail`* | Fired when a selection has been made. Attribute `index` is the selected index (will  be of type `number`) |
```

Comment:
nit: I'd use the term "Property" since the detail is an object and "Attribute" means something else

Reply:
👍